test/components/structures/auth/Registration-test.js

```diff
@@ -77,7 +84,7 @@ describe('Registration', function() {
         expect(form).toBeTruthy();
     });
 
-    it("should show SSO options if those are available", () => {
+    it("should show SSO options if those are available", async () => {
```

Comment:
These changes don't look like they belong in this PR...

Reply:
Split this into it's own PR, will rebase once it's merged